apps/cli/src/commands/environment/create.env.ts

```diff
@@ -0,0 +1,58 @@
+import BaseCommand from '../base.command'
+import Logger from '../../util/logger'
+import EnvironmentController from '../../http/project'
+import {
+  CommandActionData,
+  CommandOption
+} from 'src/types/command/command.types'
+import { EnvironmentData } from 'src/types/command/environment.types'
+
+export class CreateEnvironment extends BaseCommand {
+  private environmentController = new EnvironmentController()
+
+  getName(): string {
+    return 'create'
+  }
+
+  getDescription(): string {
+    return 'Create a new environment'
+  }
+
+  getOptions(): CommandOption[] {
+    return []
+  }
+
+  async action({ args }: CommandActionData): Promise<void> {
+    const [project_id] = args
+
+    if (!project_id) {
+      Logger.error('Project ID is required')
+      return
+    }
+
+    const baseUrl = process.env.BASE_URL
+    const apiKey = process.env.API_KEY
+
+    const environmentData: EnvironmentData = {
```

Comment:
Refer to how we parse the profile data from the function parameters in the `profile -> create` function. That will give you the idea.

Reply:
Yeah Sure Thanks a lot

Reply:
https://docs.keyshade.xyz/contributing-to-keyshade/design-of-our-code/cli#the-action-function

Docs about the `action` function

Reply:
oh that's nice will work upon it Thanks!!